chutney/server/src/main/java/com/chutneytesting/campaign/infra/DatabaseCampaignRepository.java

```diff
@@ -47,7 +47,7 @@ public DatabaseCampaignRepository(CampaignJpaRepository campaignJpaRepository,
     @Override
     @Transactional
     public Campaign createOrUpdate(Campaign campaign) {
-        if (campaign.id != null && !campaignExists(campaign.id)) {
+        if (campaign.id != null && !campaignJpaRepository.existsById(campaign.id)) {
```

Comment:
it's not exactly the same behavior, what happens if the campaign does not exist ?